src/mumble/WebFetch.cpp

```diff
@@ -35,19 +35,36 @@
 #include "Global.h"
 #include "NetworkConfig.h"
 
-WebFetch::WebFetch(QUrl url, QObject *obj, const char *slot) : QObject(), qoObject(obj), cpSlot(slot) {
-	url.setScheme(QLatin1String("http"));
+WebFetch::WebFetch(QString service, QUrl url, QObject *obj, const char *slot)
+	: QObject()
+	, qoObject(obj)
+	, cpSlot(slot)
+	, m_service(service) {
 
-	// Fix in case the regional host is broken
-	url.setHost(g.s.qsRegionalHost);
-	if (url.host() != g.s.qsRegionalHost)
-		url.setHost(QLatin1String("mumble.info"));
+	url.setScheme(QLatin1String("https"));
+
+	if (!g.s.qsServicePrefix.isEmpty()) {
+		url.setHost(prefixedServiceHost());
+	} else {
+		url.setHost(serviceHost());
+	}
 
 	qnr = Network::get(url);
 	connect(qnr, SIGNAL(finished()), this, SLOT(finished()));
 	connect(this, SIGNAL(fetched(QByteArray,QUrl,QMap<QString,QString>)), obj, slot);
 }
 
+QString WebFetch::prefixedServiceHost() const {
+	if (g.s.qsServicePrefix.isEmpty()) {
+		return serviceHost();
+	}
+	return QString::fromLatin1("%1-%2.mumble.info").arg(g.s.qsServicePrefix, m_service);
```

Comment:
Maybe we should do "%2%1" instead. That way we can decide on the service side whether we want ```publist.foo.mumble.info``` or ```publist-foo.mumble.info```. WDYT?